docs/guides/software-lifecycle/continuous-testing/README.md

```diff
@@ -1,3 +1,647 @@
+3. Pull built or packaged artifacts from repositories and deploy software release (components or single package) to a virtualized environment
+4. Test the deployed release against a set of specified system tests
+
+The diagram below illustrates this concept:
+
+```mermaid
+graph TD
+    subgraph SoftwareDevelopment[Software Components]
+        Code[Code Development]
+        UnitTests{Unit Tests Success?}
+        BuildComponents[Build Stand-alone Components]
+        PublishArtifacts[(Publish Artifacts to Repositories)]
+    end
+    subgraph IntegratedSoftware[Integrated Software]
+        SingleIntegratedBuild{Single Integrated Build?}
+        BuildIntegratedSoftware[Build Integrated Software]
+        PublishIntegratedArtifact[(Publish Integrated Artifact to Repository)]
+    end
+
+    subgraph TestDeployment[Testing & Deployment, e.g. nightly / weekly]
+        PullArtifacts[Pull Artifacts]
+        DeployRelease[Deploy Software Release]
+        IntegrationTests[Test Against System Tests]
+    end
+
+    Code --> UnitTests
+    UnitTests -->|Yes| BuildComponents
+    UnitTests -->|No| Code
+    BuildComponents --> PublishArtifacts
+    SingleIntegratedBuild -->|Yes| BuildIntegratedSoftware
+    PublishArtifacts --> BuildIntegratedSoftware
+    BuildIntegratedSoftware --> PublishIntegratedArtifact
+    PullArtifacts --> DeployRelease
+    DeployRelease --> IntegrationTests
+    PublishArtifacts --> TestDeployment
+    PublishIntegratedArtifact --> TestDeployment
+```
+
+##### Types of System Tests
+
+You should outline the types of system tests you plan to implement in your `TESTING.md` file. We suggest the following types of tests to include:
+- Testing for integration
+  - Interaction between software components
+  - Interaction with external services, provided files, exchange of messages, etc.
+- Testing for security
+  - See existing [SLIM security best practices](/slim/docs/category/security) as part of your software development workflow.
+- Testing for performance and load
+- Testing user interfaces for gaps and compliance against policies
+
+##### Integration Tests Automation
+
+We recommend the following steps for integration tests automation:
+1. Integrate your integration tests into a [Continuous Integration (CI) pipeline](/docs/guides/software-lifecycle/continuous-integration), allowing for automatic execution of tests upon code changes.
+2. Ensure that each integration test can run independently without relying on the state or output of other tests. 
+
+##### Security Tests Automation
+
+To aid in security testing automation, we recommend two steps:
+1. Add security testing to your developers' local coding environment via pre-commit (see [Unit Test Automation](#31-unit-test-automation) section above)
+2. Enable existing [SLIM security best practices](/slim/docs/category/security) as part of your software development workflow.
+
+##### Performance Tests Automation
```

Comment:
Can we point to actual recommended tools or strategies described on the web? Also - please consider pointing to the testing-frameworks.md document